### PR TITLE
Ghost Explore in Ghost

### PR DESCRIPTION
**Pull request by aileen**
no issue

- Added Ghost Explore screen behind alpha flag
- Moved existing `/explore` route to `/explore/connect` which we'll redirect to for outside requests
- Added iframe communication with Ghost Explore

### CODE REVIEW

File: ghost/admin/app/components/gh-explore-iframe.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            this.explore.getExploreIframe().contentWindow.postMessage({
+                query: 'getSiteData',
+                response: 'siteData'
+            }, '*');
+        }
+    }
+
+    _handleSiteDataUpdate(data) {
+        // TODO: now we received the site data, we can show the fireworks and the actual data
+        this.explore.siteData = data.siteData;
```
Comment (aileen) [inline ghost/admin/app/components/gh-explore-iframe.js:43]: @peterzimon This is where we receive the site data, so we can trigger confetti and the success modal with that data.

File: ghost/admin/app/components/gh-explore-iframe.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import Component from '@ember/component';
```
Comment (kevinansfield) [inline ghost/admin/app/components/gh-explore-iframe.js:1]:
```suggestion
import Component from '@glimmer/component';
```

File: ghost/admin/app/controllers/application.js [hunk 3]
```diff
@@ -19,6 +20,9 @@ export default class ApplicationController extends Controller {
     @reads('config.hostSettings.billing.enabled')
         showBilling;
 
+    @reads('explore.enabled')
+        showExplore;
+
```
[the remark this answers is not in the document]
Comment (kevinansfield) [reply ghost/admin/app/controllers/application.js:25]: Or better yet, use `{{#if (feature "exploreApp")}}...{{/if}}` in templates to match other feature flag usage.

File: ghost/admin/app/templates/application.hbs [hunk 4]
```diff
@@ -14,6 +14,9 @@
             {{#if this.showBilling}}
                 <GhBillingModal @billingWindowOpen={{this.billing.billingWindowOpen}} />
             {{/if}}
+            {{#if (feature "exploreApp")}}
+                <GhExploreModal @exploreWindowOpen={{this.explore.exploreWindowOpen}} />
```
Comment (kevinansfield) [inline ghost/admin/app/templates/application.hbs:18]: Minor thing but there's no need to pass `@exploreWindowOpen` through as an argument, the component's `visibilityClass` getter can use `this.explore.exploreWindowOpen` directly and then there's no need to inject the service on the application controller.

File: ghost/admin/app/components/gh-nav-menu/main.js [hunk 5]
```diff
@@ -15,6 +15,7 @@ import {task} from 'ember-concurrency';
 @tagName('')
 export default class Main extends Component.extend(ShortcutsMixin) {
     @service billing;
+    @service explore;
```
Comment (kevinansfield) [inline ghost/admin/app/components/gh-nav-menu/main.js:18]: This isn't used anywhere since switching to the `{{feature}}` helper 🙂

**Pull request merged**